test/css/validate.js

```diff
@@ -0,0 +1,32 @@
+const assert = require('assert').strict;
+const { definitionSyntax } = require('css-tree');
+
+const css = require('@webref/css');
+
+const cssValues = [
+  { type: 'property', prop: 'properties', value: 'value' },
+  { type: 'extended property', prop: 'properties', value: 'newValue' },
+  { type: 'descriptor', prop: 'descriptors', value: 'value' },
+  { type: 'value space', prop: 'valuespaces', value: 'value' }
+];
+
+css.parseAll().then(all => {
+  for (const [shortname, json] of Object.entries(all)) {
+    describe(`The ${shortname} entry in @webidl/css`, () => {
+      for (const { type, prop, value } of cssValues) {
+        if (!json[prop]) {
+          continue;
+        }
+        for (const [name, desc] of Object.entries(json[prop])) {
+          if (!desc[value]) {
+            continue;
+          }
+          it(`defines a valid ${type} "${name}"`, () => {
+            const ast = definitionSyntax.parse(desc[value]);
+            assert(ast.type);
```

Comment:
Is there anything more that can be asserted about what CSSTree returns?

Reply:
Probably. I haven't looked into that for now. I propose to address further improvements afterwards.

Reply:
Tracked in #127.